tardis/plasma/properties/nlte.py

```diff
@@ -86,7 +86,7 @@ def calculate(
         helium_population.loc[0, 0] = 0.0
         # He II excited states
         he_two_population = level_boltzmann_factor.loc[2, 1].mul(
-            (2 ** (-1.0))
+            (float(g.loc[2,1,0]) ** (-1.0))
         )
```

Comment:
@wkerzendorf, this seems to work (with the 'float').

Reply:
oh actually if you do `g.loc[2,1,0].values` might also work?